templates/dashboard/index.tpl

```diff
@@ -12,7 +12,7 @@
 {assign var="uuid" value=""|uniqid|escape}
 <div id="dashboard-{$uuid}">
 	<tabs>
-		<tab id="myQueue" label="{translate key="dashboard.myQueue"}">
+		<tab id="myQueue" label="{translate key="dashboard.myQueue"}" :badge="components.{$smarty.const.SUBMISSIONS_LIST_MY_QUEUE}.itemsMax">
```

Comment:
We also want the badge added for each one of the lists here (unassigned, all active, archive). You should be able to find all of the constants by searching for `SUBMISSIONS_LIST_` in the code base.